admin/tabs/activity/clienterrors/ClientErrorModel.js

```diff
@@ -29,6 +29,7 @@ export class ClientErrorModel {
     gridModel = new GridModel({
         stateModel: 'xhClientErrorGrid',
         enableColChooser: true,
+        exportFilename: () => `Client Errors ${fmtDate(this.startDate)} to ${fmtDate(this.endDate)}`,
```

Comment:
interesting that export is enabled by default.  Given the number of things that have to be done to make an export "look good" should we make it opt-in?